style.css

```diff
@@ -1,4 +1,21 @@
 /* Write your CSS here! */
+.header {
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+}
+
+.primary-content {
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+}
+
+.footer {
```

Comment:
It's possible to achieve what you were asked to do without using the position fixed. try to come back to this if you have some time.